docs/src/components/home/DesignSystemComponents.tsx

```diff
@@ -0,0 +1,268 @@
+        marginTop: -1,
+      },
+    },
+    [theme.breakpoints.only('sm')]: {
+      marginTop: -1,
+      '&:first-of-type': {
+        borderTopLeftRadius: theme.shape.borderRadius,
+      },
+      '&:last-of-type': {
+        borderBottomRightRadius: theme.shape.borderRadius,
+        borderStyle: 'dashed',
+      },
+      '&:nth-of-type(even)': {
+        marginLeft: -1,
+      },
+      '&:nth-last-of-type(2)': {
+        borderBottomLeftRadius: theme.shape.borderRadius,
+      },
+      '&:nth-of-type(2)': {
+        borderTopRightRadius: theme.shape.borderRadius,
+      },
+    },
+    [theme.breakpoints.up('md')]: {
+      marginTop: -1,
+      '&:not(:nth-of-type(3n + 1))': {
+        marginLeft: -1,
+      },
+      '&:first-of-type': {
+        borderTopLeftRadius: theme.shape.borderRadius,
+      },
+      '&:last-of-type': {
+        borderBottomRightRadius: theme.shape.borderRadius,
+        borderStyle: 'dashed',
+      },
+      '&:nth-last-of-type(3)': {
+        borderBottomLeftRadius: theme.shape.borderRadius,
+      },
+      '&:nth-of-type(3)': {
+        borderTopRightRadius: theme.shape.borderRadius,
+      },
+    },
+  },
+}));
+
+function Demo({
+  name,
+  children,
+  control,
+}: {
+  name: string;
+  children: React.ReactElement;
+  control?: { prop: string; values: Array<string> };
+}) {
+  const [propValue, setPropValue] = React.useState(control ? control.values[0] : '');
+  return (
+    <Box sx={{ display: 'flex', flexDirection: 'column', height: '100%' }}>
+      {control ? (
+        <Box sx={{ minHeight: 40, ml: -1, mt: -1 }}>
+          <Tabs
+            value={propValue}
+            onChange={(event, value) => setPropValue(value)}
+            sx={{
+              minHeight: 'initial',
+              '& .MuiTabs-indicator': {
+                bgcolor: 'transparent',
+                '&:before': {
+                  height: '100%',
+                  content: '""',
+                  display: 'block',
+                  width: (theme) => `calc(100% - ${theme.spacing(2)})`,
+                  bgcolor: 'primary.main',
+                  position: 'absolute',
+                  top: 0,
+                  left: (theme) => theme.spacing(1),
+                },
+              },
+              '& .MuiTab-root': {
+                px: 1,
+                pt: 0.5,
+                minWidth: 'initial',
+                minHeight: 'initial',
+                fontWeight: 'medium',
+              },
+            }}
+          >
+            {control.values.map((value) => (
+              <Tab value={value} label={value} />
+            ))}
+          </Tabs>
+        </Box>
+      ) : null}
+      <Box sx={{ flexGrow: 1, display: 'flex', alignItems: 'center', justifyContent: 'center' }}>
+        {React.cloneElement(children, {
+          ...(control && {
+            [control.prop]: propValue,
+          }),
+        })}
+      </Box>
+      <Typography fontWeight="bold">{name}</Typography>
```

Comment:
Reminder for changing this Typography to the variant that is 14px (couldn't find where it is defined).

Reply:
Just checked, I need to go over the typography again because I follow the size in figma and it turns out that it is different from webflow. That's fine, just a quick fix.

Reply:
I tired to make them to be equal. So, if theres something unsynced, it might be an one-off, so I'd suggest looking for the pattern. Additionally, Webflow will probably be more trustable. You can always check back at Figma though what variant a given arbitrary value stands for, when inspecting a Webflow demo, for referencing: text-sm -> 14px, text-md -> 16px, etc.